src/validator.cpp

```diff
@@ -229,9 +236,10 @@ struct Validator::ValidatorImpl
      *
      * @param index The index of the @c unit to validate from @p units.
      * @param units The units to validate.
-     * @param unitsNames A vector list of the name attributes of the @p units and its siblings.
+     * @param history The history of units visited.
+     * @param modelsVisited The list of visited models.
      */
-    void validateUnitsUnit(size_t index, const UnitsPtr &units, const std::vector<std::string> &unitsNames) const;
+    void validateUnitsUnit(size_t index, const UnitsPtr &units, History &history, const std::vector<ModelPtr> &modelsVisited) const;
```

Comment:
I imagine this method should be renamed to `validateUnitsUnitsItem()`?

Reply:
2d559a7